docs/docs/model-registry/index.mdx

```diff
@@ -2,6 +2,947 @@
 sidebar_position: 15
 ---
 
+import { APILink } from "@site/src/components/APILink";
+import TOCInline from "@theme/TOCInline";
+import Link from "@docusaurus/Link";
+
 # MLflow Model Registry
 
-{/* WIP */}
+The MLflow Model Registry component is a centralized model store, set of APIs, and UI, to
+collaboratively manage the full lifecycle of an MLflow Model. It provides model lineage (which
+MLflow experiment and run produced the model), model versioning, model aliasing, model tagging, and
+annotations.
+
+<TOCInline toc={toc} maxHeadingLevel={4} minHeadingLevel={2} />
```

Comment:
let's remove this, now we have the right sidebar so it's probably not necessary. i think it's fine for some pages where it's small, but this one is huge